### ISSUE TITLE
Prevent death also apply to AI even if the medical setting is on player only.

### ISSUE THREAD

**Issue by alganthe** [acemod/ACE3#838]
ACE3 Version: 2015-04-24: build around 14h UTC+1

**Mods:**
* @cba_a3
* @ace3

**Placed ACE3 Modules:**
Medical settings:
```
Medical level: Basic
Enabled for: Players only
Enable advanced wounds: No
Vehicle crashes: Yes
Player damage: 1
AI damage: 1
AI unconsciousness DISABLED
Prevent instant death YES
Bleeding coef: 1
Sync status: YES
```

**Description:**
AI also get the prevent instant death effects.

**Steps to reproduce:**
place the module as indicated previously
place a player NATO rifleman, and put "This setCaptive true;" just to avoid getting shot
place a CSAT rifleman
shoot the baddie and notice he's just unconscious from a headshot. 

**Where did the issue occur?**
SP, MP both local and non locals units are affected

**RPT log file:**
nothin' poppin' 

:banana:

**Comment by Glowbal** [comment]
Enabled for is at the moment an advanced medical only setting. It means; enable advanced medical for players or players and AI. When using players only, it will use basic medical for AI. 

We have yet to solve some things around disabling the medical system, due to the fact we have changed the hitpoints on the units.

**Comment by alganthe** [comment]
fixed in #861
just tested,thanks for the fix glowbal
